fast64_internal/mk64/mk64_actors.py

```diff
@@ -0,0 +1,200 @@
+import math, bpy, mathutils
+import os
+from bpy.utils import register_class, unregister_class
+from ..panels import MK64_Panel
+
+
+
+enumModelIDs = [
+    ("Custom", "Custom", "Custom"),
+    ("ITEMBOX", "itembox", "itembox"),
+    ("FOLIAGE", "foliage", "foliage"),
+    ("FALLING_ROCK", "fallingRocks", "fallingRocks"),
+    ("PALM_TREES", "palmTrees", "palmTrees"),
+    ("PIRANHA_PLANT", "piranhaPlant", "piranhaPlant"),
+    ("Cow", "cow", "cow"),
```

Comment:
why is this not uppercase like the other ones? (I mean the first element of that tuple), is it just because it's done like that in decomp?